Library/AnsiC/ansic.gp

```diff
@@ -156,24 +156,27 @@ export fscanf	as _fscanf
 
 incminor
 
-publish F_LDIV@
-publish F_LMOD@
-publish F_LUDIV@
-publish F_LUMOD@
-publish F_LXLSH@
-publish F_LXMUL@
-publish F_LXRSH@
-publish F_LXURSH@
-publish F_PADA@
-publish F_PADD@
-publish F_PCMP@
-publish F_PDEA@
-publish F_PINA@
-publish F_PSBA@
-publish F_PSBP@
-publish F_PSUB@
-publish F_SCOPY@
-publish F_SPUSH@

```

Comment:
Not sure how this is with publish but think every export/publish line from part of the libraries external contract/protocol, so just removing breakes the binary interface of the library. We should consider keep the slots empty instead of removing them.